### DISCUSSION TITLE
Terms of use

### DISCUSSION THREAD

**Discussion by Osamakanaan** [wilfredinni/python-cheatsheet#79] [Q&A] [answered]
**Is this website pythoncheatsheet.org free to use for commerical use?**
I am designing a python course and I need sometimes to revise my knoweldge, things like data types and more.. I copy some of them

**Comment by wilfredinni** [comment, marked as answer]
Hi! Yes, pythoncheatsheet.org is under the [MIT License](https://github.com/wilfredinni/python-cheatsheet/blob/master/LICENCE).

![image](https://user-images.githubusercontent.com/23016174/180824931-55678e26-df1f-41f9-8940-7f9272cf4209.png)

**Reply by Osamakanaan** [reply]
Do I have to keep the copied, modified or added content under the same license, or I can use any terms I wish?

**Reply by wilfredinni** [reply]
You can change the license, nothing wrong with that. But, if you find the content useful, it would be really cool of you to add a link to pythoncheatsheet.org somewhere, it will help me and this project a lot.
